roles/open_zaak_k8s/tasks/ingress_routes.yml

```
@@ -1,54 +1,11 @@
 ---
 
-# Install the Traefik ingress route(s)
-
-# HTTP
-
-- name: Install HTTP Ingress Route
-  k8s:
-    state: present
-    name: openzaak-http
-    namespace: "{{ openzaak_namespace }}"
-    definition:
-      apiVersion: traefik.containo.us/v1alpha1
-      kind: IngressRoute
-      spec:
-        entryPoints:
-          - web
-        routes:
-        - match: Host(`{{ openzaak_domain }}`)
-          kind: Rule
-          services:
-          - name: openzaak-nginx
-            port: 80
-          middlewares:
-            - name: https-redirect
-              namespace: default
-
-# HTTPS
-
-- name: Install HTTPS Ingress Route
+- name: Install Ingress Route
   k8s:
     state: present
-    name: openzaak-https
+    name: openzaak
     namespace: "{{ openzaak_namespace }}"
-    definition:
-      apiVersion: traefik.containo.us/v1alpha1
-      kind: IngressRoute
-      spec:
-        entryPoints:
-          - websecure
-        routes:
-        - match: Host(`{{ openzaak_domain }}`)
-          kind: Rule
-          services:
-          - name: openzaak-nginx
-            port: 80
-          middlewares:
-            - name: https-hsts
-              namespace: default
-        tls:
-          certResolver: default
-          options:
-            namespace: default
-            name: default
+    definition: "{{ lookup('template', 'ingress.yml.j2') | from_yaml }}"
```

maybe make this template name a variable, so that people on older k8s versions can provide their own template:

```suggestion
    definition: "{{ lookup('template', openzaak_ingress_tpl|default('ingress.yml.j2')) | from_yaml }}"
```

I'd also rename this file from `ingress_routes.yml` to `ingress.yml` to cover the meaning better.

---

Done.